static/app/gettingStartedDocs/javascript/angular.tsx

```diff
@@ -149,7 +149,12 @@ bootstrapApplication(appConfig, AppComponent)
 
   const config = buildSdkConfig({
     params,
-    staticParts: [`dsn: "${params.dsn.public}"`],
+    staticParts: [
+      `dsn: "${params.dsn.public}"`,
+      `// Setting this option to true will send default PII data to Sentry.
+      // For example, automatic IP address collection on events
+      sendDefaultPii: true`,
```

Comment:
since we have the same text + property everywhere, what do you think about creating a reusable const and just reuse it everywhere where we need?

Reply:
it's just easier effort to not do it, but I am not against. Was more just interested in shipping this as a quick fix. Later we need to add docs reference. Maybe then someone can look into abstracting this more

for now i think it's scope creep